api-reference/10 UI Components/dxScheduler/1 Configuration/onAppointmentFormOpening.md

```diff
@@ -33,7 +33,10 @@ Model data. Available only if you use Knockout.
 The instance of the popup that contains the form.
 
 ---
+<<<<<<< HEAD
 The appointment details form is displayed inside a popup. These elements are implemented by the [Form](/concepts/05%20UI%20Components/Form/00%20Overview.md '/Documentation/Guide/UI_Components/Form/Overview/') and [Popup](/concepts/05%20UI%20Components/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI components. Their instances are passed to the **onAppointmentFormOpening** function in the **form** and **popup** fields. Use the [Form API](/api-reference/10%20UI%20Components/dxForm/1%20Configuration '/Documentation/ApiReference/UI_Components/dxForm/Configuration/') and [Popup API](/api-reference/10%20UI%20Components/dxPopup/1%20Configuration '/Documentation/ApiReference/UI_Components/dxPopup/Configuration/') to customize the corresponding UI component.
+=======
+>>>>>>> b73274fb3... Prepare Getting Started with Popup Guide (#2137)
```

Comment:
Merge conflict was resolved incorrectly. The link to Popup Overview should be replaced.